Wins tests fail only if e2e tests fail

/kind bug

Windosw tests on IBM Cloud fail only if e2e tests fail.

If integration tests fail, the complete test pass.


```
[...]
Summarizing 1 Failure:
  [FAIL] odo init interactive command tests [It] should print automation command with proper values [nocluster]
  C:/Users/Administrator.ANSIBLE-TEST-VS/2586/tests/integration/interactive_init_test.go:123

Ran 91 of 427 Specs in 18.435 seconds
FAIL! -- 90 Passed | 1 Failed | 0 Pending | 336 Skipped


Ginkgo ran 1 suite in 42.7933764s

Test Suite Failed
exit status 1
make: *** [Makefile:197: test-integration-no-cluster] Error 1
[...]

Ran 326 of 427 Specs in 994.369 seconds
SUCCESS! -- 326 Passed | 0 Failed | 0 Pending | 101 Skipped


Ginkgo ran 1 suite in 16m47.6261506s
Test Suite Passed

[...]

Ran 4 of 4 Specs in 138.250 seconds
SUCCESS! -- 4 Passed | 0 Failed | 0 Pending | 0 Skipped


Ginkgo ran 1 suite in 2m25.1988222s
Test Suite Passed

[...]

Finished: SUCCESS
```


## What versions of software are you using?

**Operating System:**

**Output of `odo version`:**

## How did you run odo exactly?


## Actual behavior


## Expected behavior


## Any logs, error output, etc?

--- a/.ibm/pipelines/windows-test-script.ps1
+++ b/.ibm/pipelines/windows-test-script.ps1
@@ -81,6 +81,7 @@ function Run-Test {
     make install 
     Shout "Running test"
     make test-integration-no-cluster | tee -a  C:\Users\Administrator.ANSIBLE-TEST-VS\AppData\Local\Temp\$LOGFILE
+    Check-ExitCode $LASTEXITCODE
     make test-integration-cluster    | tee -a  C:\Users\Administrator.ANSIBLE-TEST-VS\AppData\Local\Temp\$LOGFILE
     Check-ExitCode $LASTEXITCODE
     make test-e2e                    | tee -a  C:\Users\Administrator.ANSIBLE-TEST-VS\AppData\Local\Temp\$LOGFILE

